test/run_test.py

```diff
@@ -0,0 +1,245 @@
+    if child.returncode:
+        fatal_error("Following error was generated by gcore:", stderr)
+
+    # get only last line form output - there is filename mentioned
+    lastline = stdout.splitlines()[-1]
+    # get the last word, that should be coredump path
+    corepath = lastline[lastline.rfind(" ")+1:]
+    if not corepath.startswith("/var/tmp"):
+        fatal_error("Corefile was not generated")
+
+    # rename it, so it is called coredump, not coredump.XXXXX
+    os.rename(corepath, corepath[:corepath.find(".")])
+    corepath = corepath[:corepath.find(".")]
+
+    # find in which package is python
+    exe_link = os.readlink('/proc/self/exe')
+    rpm_cmdline = 'rpm -qf '+exe_link
+    child = Popen(rpm_cmdline, shell=True, stdout=PIPE, stderr=PIPE)
+    package, stderr = child.communicate()
+    if child.returncode:
+        fatal_error("rpm for python was not found")
+    package = package[:package.rfind(".")]
+    return (corepath, package, sys.executable)
+
+
+def create_local_config(change_repo_dir):
+    """Create local config file, change what is needed and edit env var."""
+    # create local copy of set configuration file
+    # read current configuration file
+    config_path = os.environ.get('RETRACE_SERVER_CONFIG_PATH')
+    new_conf_path = '/var/tmp/retrace-server.conf'
+
+    with open(config_path) as org_file:
+        with open(new_conf_path, "w") as new_file:
+            for line in org_file:
+                if line.startswith("RepoDir") and not change_repo_dir:
+                    line = 'RepoDir = /var/tmp\n'
+                if line.startswith("RequireGPGCheck"):
+                    line = 'RequireGPGCheck = 0\n'
+                new_file.write(line)
+
+    # edit env variable RETRACE_SERVER_CONFIG_PATH
+    os.environ["RETRACE_SERVER_CONFIG_PATH"] = new_conf_path
+
+
+# Parse arguments
+parser = argparse.ArgumentParser(description="Run test for retrace-server.")
+parser.add_argument("--coredump", help="path to the coredump, if not \
+                                       specified, will be generated")
+parser.add_argument("--release", help="os_release,\
+                                      default=Fedora release 24 (Twenty Four)")
+parser.add_argument("--executable", help="path to the executable")
+parser.add_argument("--package", help="name of package")
+parser.add_argument("--delete_repo", help="remove repo after bt generation",
+                    action="store_true")
+parser.add_argument("--dont_create_repo", help="don't create repo before bt\
+                    generation", action="store_true")
+parser.add_argument("--interactive", help="run as interactive task, that means\
+                    that task will not be deleted at the end.",
+                    action="store_true")
+args = parser.parse_args()
+
+# set default values
+os_release = args.release if args.release else "Fedora release 24 (Twenty Four)"
+dont_create_repo_arg = True if args.dont_create_repo else False
```

Comment:
I am wondering why you cannot just assign `args.dont_create_repo` to `dont_create_repot`.